flink-runtime/src/main/java/org/apache/flink/runtime/deployment/PartialInputChannelDeploymentDescriptor.java

```diff
@@ -44,15 +44,15 @@
 	private final ResultPartitionID partitionID;
 
 	/** The partition connection info. */
-	private final InstanceConnectionInfo partitionConnectionInfo;
+	private final TaskManagerLocation partitionConnectionInfo;
 
 	/** The partition connection index. */
 	private final int partitionConnectionIndex;
 
 	public PartialInputChannelDeploymentDescriptor(
 			IntermediateDataSetID resultId,
 			ResultPartitionID partitionID,
-			InstanceConnectionInfo partitionConnectionInfo,
+			TaskManagerLocation partitionConnectionInfo,
```

Comment:
Shall we adapt the variable names as well? Otherwise it might be a little confusing that a `partitionConnectionInfo` is actually of type `TaskManagerLocation`.